### PR TITLE
Update and embed rook/ceph into provider images

### PR DESCRIPTION
**Pull request by mhenriks**
Signed-off-by: Michael Henriksen <mhenriks@redhat.com>

Basically just update of embedded ceph.  Now managed by rook rather than straight ceph-csi.

Ceph now requires block devices provisioned on the host.  So we do that via `--block-device` and `--block-device-size` params to vm.sh.

### CODE REVIEW

File: cluster-provision/gocli/cmd/run.go [hunk 1]
```diff
@@ -488,6 +457,11 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 			nodeQemuArgs = "--qemu-args \"" + nodeQemuArgs + "\""
 		}
 
+		blockDev := ""
+		if cephEnabled {
+			blockDev = "--block-device /var/run/disk/blockdev.qcow2 --block-device-size 32212254720"
```
[the remark this answers is not in the document]
Comment (mhenriks) [reply cluster-provision/gocli/cmd/run.go:462]: Not sure what environment variable you mean but these are params passed to the vm.sh script.  block device params aren't user set-able (added to gocli).

[the remark this answers is not in the document]
Comment (mhenriks) [reply cluster-provision/gocli/cmd/run.go:462]: I could potentially see block device size being user set but not the path

[the remark this answers is not in the document]
Comment (awels) [reply cluster-provision/gocli/cmd/run.go:462]: I mean BLOCK_DEV_SIZE set in vm.sh, I was assuming that would be passed to --block-device-size $BLOCK_DEV_SIZE but maybe I am missing something

[the remark this answers is not in the document]
Comment (awels) [reply cluster-provision/gocli/cmd/run.go:462]: Oh I got it reversed, we are passing that into vm.sh, gotcha

**Review by awels — COMMENTED** [review]
/lgtm

**Comment by maya-r** [comment]
@rmohr @davidvossel @slintes @vladikr -- this PR only needs an approval to move forward, could you have a look at it? thanks.

**Pull request merged**